### DISCUSSION TITLE
在第三阶段train_prompts.sh脚本中有命令torchrun --standalone --nproc_per_node=1 train_prompts.py prompts.csv --strategy colossalai_zero2

### DISCUSSION THREAD

**Discussion by zhixingheyier** [hpcaitech/ColossalAI#3380] [Community | Q&A]
请问prompts.csv文件在什么位置？

**Comment by binmakeswell** [comment]
@zhixingheyier It means your data file, the json file at https://github.com/hpcaitech/ColossalAI/tree/main/applications/Chat/examples#supervised-datasets-collection

**Reply by zhixingheyier** [reply]
thanks for your reply, it remain some wrong when I had downloaded the .json file and run the train_prompts.sh.
![image](https://user-images.githubusercontent.com/66710115/230378973-ac4e6a02-6a42-4ee8-bbf7-6b4f69280a25.png)
![image](https://user-images.githubusercontent.com/66710115/230378844-afc971cb-83fd-4008-8959-8529eb11ccaa.png)
